internal/pkg/policy/policy_output.go

```diff
@@ -17,16 +17,19 @@ import (
 
 	"github.com/elastic/fleet-server/v7/internal/pkg/apikey"
 	"github.com/elastic/fleet-server/v7/internal/pkg/bulk"
+	"github.com/elastic/fleet-server/v7/internal/pkg/config"
 	"github.com/elastic/fleet-server/v7/internal/pkg/dl"
+	"github.com/elastic/fleet-server/v7/internal/pkg/es"
 	"github.com/elastic/fleet-server/v7/internal/pkg/logger"
 	"github.com/elastic/fleet-server/v7/internal/pkg/model"
 	"github.com/elastic/fleet-server/v7/internal/pkg/smap"
 )
 
 const (
-	OutputTypeElasticsearch = "elasticsearch"
-	OutputTypeLogstash      = "logstash"
-	OutputTypeKafka         = "kafka"
+	OutputTypeElasticsearch       = "elasticsearch"
+	OutputTypeRemoteElasticsearch = "remote-elasticsearch"
```

Comment:
I'm wondering if creating a new type is the best option. Ins't a ES output always an ES output. I mean, the `elasticsearch` and `remote-elasticsearch` have exactly the same data?

Without much thought I'd say the distinction should be made either on the output name, or extending the model for the ES output. I prefer the distinction being based on the output name.

Reply:
The config is different slightly as the remote-elasticsearch has a service_token, and the different type is used on the UI to render different components. I was thinking about it though if it would simplify to use the same type

Reply:
> The config is different slightly as the remote-elasticsearch has a service_token
> [...]
> so fleet-server doesn't have to introduce a new type, it can rely on the presense of the `service_token` field.

I believe relying on the presence of `service_token` isn't ideal. Besides if they're indeed different, then having 2 types makes sense. I thought at first they'd be the same thing, the new type was only to differentiate between the fleet-ES and the remote-ES

Reply:
Okay, it makes sense. I still have to change the type back to elasticsearch before sending it to the agent, as the agent doesn't know anything about "remote-elasticsearch" output type, and doesn't need to.

Reply:
that's true! how could I forget that...
in that case I'd strongly suggest sticking with only `elasticsearch` as type and differentiate them some other way